pinot-integration-tests/src/test/java/org/apache/pinot/compat/tests/TableOp.java

```diff
@@ -92,4 +102,34 @@ boolean runOp() {
     }
     return true;
   }
+
+  private boolean createSchema() {
+    try {
+      RequestProcessor.sendPostRequestWithFile(CONTROLLER_URL + "/schemas", new File(_schemaFileName));
```

Comment:
You can use `ControllerRequestURLBuilder.baseUrl(CONTROLLER_URL).forSchemaCreate()`

Same for table creation/deletion, the constructor should all inside `ControllerRequestURLBuilder` (If not, please add it there).